fretboard/src/main/java/mozilla/components/service/fretboard/Fretboard.kt

```diff
@@ -4,4 +4,37 @@
 
 package mozilla.components.service.fretboard
 
-class Fretboard
+/**
+ * Entry point of the library
+ *
+ * @param source experiment remote source
+ * @param storage experiment local storage mechanism
+ */
+class Fretboard(
+    private val source: ExperimentSource,
+    private val storage: ExperimentStorage
+) {
+    internal var experiments: List<Experiment> = listOf()
+        private set
```

Comment:
Let's make the whole thing private for now. We probably do not want consumers to access the list directly.

Reply:
It was meant for testing, but it's right it seems better as private all the times.